data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg

```diff
@@ -406,7 +406,7 @@
 
                 [message]
                     speaker="Ro'Sothian"
-                    message= _ "Camerin! Great Lords of Darkness, I haven’t seen you in ages! Here, let me just finish off this little vermin and I’ll be right over."
+                    message= _ "Camerin! Great Lords of Darkness, I haven’t seen you in ages! Here, let me just finish off this little vermin, and I’ll be right over."
```

Comment:
I prefer the original, the lack of pause matches the time that he's expecting the action to take.

Reply:
Okay please revert to original.